compiler/quilt/tools/command.py

```diff
@@ -1129,3 +1129,19 @@ def _print_node(node, prefix, child_prefix, name, path):
 
     print(pkgobj.get_path())
     _print_children(children=pkgobj.get_contents().children.items(), prefix='', path='')
+
+def rm(package, force=False):
+    """
+    Remove a package (all instances) from the local store.
+    """
+    owner, pkg = parse_package(package)
+
+    if not force:
+        confirmed = input("Remove {0}? (y/n)".format(package))
+        if confirmed.lower() != 'y':
+            return
+
+    store = PackageStore()
+    deleted = store.remove_package(owner, pkg)
+    for obj in deleted:
+        print("Removing: {0}".format(obj))
```

Comment:
I think it's kind of weird that `print` happens after the objects get deleted. At first, I actually thought you forgot to call `os.remove` here. What's wrong with printing in the `prune` method?

Reply:
We're trying to separate the API from interactive commands so it seems better to move the printing to command.py. I changed "removing" to "removed" in the output to make it clear that it already happened.